### PR TITLE
Use WellContainer<double> for bhp/thp/temperature

### PR DESCRIPTION
**Pull request by joakim-hove**

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this please

File: opm/simulators/wells/WellState.cpp [hunk 1]
```diff
@@ -102,12 +99,15 @@ void WellState::initSingleWell(const std::vector<double>& cellPressures,
...
-        // No perforations of the well. Initialize to zero.
-        bhp_[w] = 0.;
-        thp_[w] = 0.;
+    this->bhp_.add(well.name(), 0.0);
+    this->thp_.add(well.name(), 0.0);
+    if ( well.isInjector() )
+        this->temperature_.add(well.name(), well.injectionControls(summary_state).temperature);
+    else
+        this->temperature_.add(well.name(), 273.15 + 15.56); // standard condition temperature
+
```
Comment (blattms) [inline opm/simulators/wells/WellState.cpp:108]: Why wasn't this needed before?

Comment (joakim-hove) [reply opm/simulators/wells/WellState.cpp:108]: The well temperature has always been initialized to 273.15 + 15.56  - no change here.

**Comment by joakim-hove** [comment]
jenkins build this please

**Pull request merged**